.github/workflows/deploy-stg-ecs.yml

```diff
@@ -46,6 +47,8 @@ jobs:
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           NPM_BASE_64_AUTH: ${{ secrets.NPM_BASE_64_AUTH }}
           NPM_EMAIL: ${{ secrets.NPM_EMAIL }}
+          # TODO: revert comment. testing flag here
+          # USE_PERSISTENCE: true
```

Comment:
I think we'll want to get rid of these occurrences since we don't need to pipe the flag in here anymore 😄